MapboxCoreNavigation/MBFixLocation.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import MapboxNavigationNative
+import CoreLocation
+
+extension MBFixLocation {
+    
+    func asCLLocation() -> CLLocation {
```

Comment:
Would it make more sense to make the return here optional? How would we know if the location is bogus values fell back to zero?